api/types/container/container.go

```diff
@@ -167,7 +167,7 @@ type InspectBase struct {
 // InspectResponse is the response for the GET "/containers/{name:.*}/json"
 // endpoint.
 type InspectResponse struct {
-	*InspectBase
+	*ContainerJSONBase
```

Comment:
Couldn't this also have a tag instead? ``*InspectBase `json:"ContainerJSONBase"``?

(Is this really embedded into the JSON as `"ContainerJSONBase": ...`??? :sob:)

Reply:
If not, I don't actually understand how that PR broke this nor how this PR fixes it. :see_no_evil: